captain/navigation.go

```diff
@@ -73,24 +63,15 @@ managing:
 				"spn:connected-to-home-hub",
 				"Connected to SPN Test Network",
 				fmt.Sprintf("You are connected to the SPN test network at %s. This notification is persistent for awareness.", homeTerminal.RemoteAddr()),
```

Comment:
I am unsure if we should call it the "SPN Test Network" - why not only mention the SPN? The SPN is in the "Broader Testing" phase, which is disclaimed on all our announcements.

> You are connected to the SPN at %s...